paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -126,26 +126,20 @@ internal fun PaymentMethodUI(
     modifier: Modifier = Modifier,
     onItemSelectedListener: (Int) -> Unit
 ) {
-    val strokeColor = colorResource(
+    val strokeColor =
         if (isSelected) {
-            R.color.stripe_paymentsheet_add_pm_card_selected_stroke
+            PaymentsTheme.colors.material.primary
         } else {
-            R.color.stripe_paymentsheet_add_pm_card_unselected_stroke
+            PaymentsTheme.colors.colorComponentBorder
         }
```

Comment:
This pattern is duplicated in a lot of places, I wonder if a helper funciton on the theme would prevent having to update it in a lot of places in the case of a change.